moveit_experimental/moveit_jog_arm/CMakeLists.txt

```diff
@@ -63,6 +63,13 @@ install(
 
 install(DIRECTORY include/ DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION})
 
+# to run: catkin_make -DENABLE_COVERAGE_TESTING=ON package_name_coverage
```

Comment:
Please remove the comment. It doesn't apply (anymore).

Reply:
done